tecken/settings.py

```diff
@@ -68,6 +68,7 @@ def dict_parser(val):
         ("OIDC_OP_TOKEN_ENDPOINT", "http://example.com/"),
         ("OIDC_OP_USER_ENDPOINT", "http://example.com/"),
         ("DATABASE_URL", "postgresql://postgres:postgres@db/tecken"),
+        ("REDIS_URL", "redis://redis-cache:6379/0"),
```

Comment:
This set of settings is gross, but it's the easiest way to make sure `./manage.py collectstatic` works when building the container in a context where there is no configuration.

Because I removed the default for `REDIS_URL`, we need to add this line here.